server/sonar-server/src/main/java/org/sonar/server/computation/ReportProcessingTask.java

```diff
@@ -59,7 +59,7 @@ public void run() {
 
     ComputeEngineContainer computeEngineContainer = containerFactory.create(sqContainer, item);
     try {
-      computeEngineContainer.process();
+      computeEngineContainer.getComponentByType(ReportProcessor.class).process();
     } catch (Throwable e) {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Catch Exception instead of Throwable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1181)